### ISSUE TITLE
Willing to donate for this awesome addon to work for Dragonflight

### ISSUE THREAD

**Issue by ZynnlolGaming** [haste/oGlow#24]
Today dragonflight updated prepatch and it broke this godlike addon, like title said ... much love for this addon !!

I know its a long shot but been using this since forever

**Comment by Tealc-1899** [comment]
Absolutely! I still use and enjoy this addon, have used it since original WotLK days.

Unfortunately I have also discovered the same issue that it has stoped working this week with the Dragonflight pre-patch. Am happy to post errors if it will help.

Hopefully this still useful addon can be updated and made to work again. I would really appreciate any efforts that can be made. Thanks.
